docs/azure/sdk/includes/managed-identity-concepts.md

```diff
@@ -0,0 +1,10 @@
+## Essential managed identity concepts
+
+A managed identity enables your app to securely connect to other Azure resources without the use of secret keys or other application secrets. Internally, Azure tracks the identity and which resources it's allowed to connect to. Azure uses this information to automatically obtain Microsoft Entra tokens for the app to allow it to connect to other Azure resources.
```

Comment:
Somewhere in this section, I'd expect to see a link to https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/managed-identity-best-practice-recommendations#choosing-system-or-user-assigned-managed-identities.

Reply:
> Somewhere in this section, I'd expect to see a link to https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/managed-identity-best-practice-recommendations#choosing-system-or-user-assigned-managed-identities.

I was considering whether this whole section should be split out into its own intro/overview doc in the auth section, or added to the main overview. However, as your link shows, there's already a lot of content out there about this. We could potentially add a contextual link to that doc in the ToC as an intro/overview, but that might be too much. For now I can just highlight that link, just let me know if you have any other thoughts around this.